joomla/joomla-cms
# 4.0 rename language files

This PR renames the language files so they no longer have a language code prefix (eg en-GB.com_content.ini becomes just com_content.ini).
This is based on a change made in 3.10: https://github.com/joomla/joomla-cms/pull/19353

### Summary of Changes
* Removes language prefix from language files
* The files named `en-GB.ini` become `joomla.ini`
* The files named `en-GB.xml` become `langmetadata.xml`


### Testing Instructions
Enable language debug so you see which files are loaded and if there are translation errors.


### Expected result
There should be no errors. If there are, they should be the same regardless of this PR.


### Documentation Changes Required
In 3.10 and 4.0, the files with language prefix will still work. But documentation for language packs and extensions should be adjusted to let them know of that change.
